src/Take.Blip.Builder/FlowManager.cs

```diff
@@ -150,6 +170,13 @@ public async Task ProcessInputAsync(Message message, Flow flow, CancellationToke
                         var stateId = await _stateManager.GetStateIdAsync(context, linkedCts.Token);
                         state = flow.States.FirstOrDefault(s => s.Id == stateId) ?? flow.States.Single(s => s.Root);
 
+                        // Cancel Schedule expiration time if input is configured
+                        if (state.Input != null &&
+                            state.Input?.IsExpirationTimeEnabled() == true)
```

Comment:
This check is duplicated below (line 287). It would be nicer to extract an extension method for `Input` which will check for null (inside it) and then check `IsExpirationTimeEnabled`...